script/playercomponent.js

```diff
@@ -214,9 +216,10 @@ define(
         var hasNextCDN = mediaMetaData.urls.length > 1;
         var aboutToEndVod = getDuration() > 0 && (getDuration() - getCurrentTime()) <= 5;
         var canVodFailover = windowType === WindowTypes.STATIC && !aboutToEndVod;
-        var canLiveFailover = windowType !== WindowTypes.STATIC && transferFormat === 'dash';
+        var canHlsLiveFailover = windowType === WindowTypes.GROWING && liveSupport === LiveSupport.SEEKABLE;
```

Comment:
Do we need to check for transfer format here as well? because this will come back true for dash

Reply:
It's probably worth noting as future changes could break this but currently, if `windowType === GROWING`, then `windowType !== STATIC` and it would pass the DASH Check anyway